### PR TITLE
Add NSUrlSessionHandler docs

### PR DESCRIPTION
**Pull request by migueldeicaza**
## Summary

NSUrlSessionHandler documentation

### CODE REVIEW

File: xml/System.Net.Http/NSUrlSessionHandler.xml [hunk 1]
```diff
@@ -146,12 +152,12 @@
...
-        <param name="request">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="request">The HTTP request message to send.</param>
+        <param name="cancellationToken">The cancellation token to cancel the operation.</param>
+        <summary>Sends an HTTP request as an asynchronous operation.</summary>
+        <returns>The task object representing the asynchronous operation.</returns>
+        <remarks>This operation is does not block. It returns an instance of <see cref="T:System.Threading.Tasks.Task" /> to represent the asynchronous operation. When the operation completes, <see cref="P:System.Threading.Tasks.Task.Result" />contains the response message.</remarks>
```
Comment (rpetrusha) [inline xml/System.Net.Http/NSUrlSessionHandler.xml]:
```suggestion
        <remarks>This operation is does not block. It returns an instance of <see cref="T:System.Threading.Tasks.Task" /> to represent the asynchronous operation. When the operation completes, <see cref="P:System.Threading.Tasks.Task`1.Result" /> contains the response message.</remarks>
```

**Comment by mairaw** [comment]
@migueldeicaza friendly ping. You should be able to batch all the suggestions into a single commit via the Files changed tab (https://github.com/dotnet/dotnet-api-docs/pull/2783/files).

File: xml/System.Net.Http/NSUrlSessionHandler.xml [hunk 2]
```diff
@@ -51,7 +57,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Allows automatic redirection, this defaults to true.</summary>
```
[the remark this answers is not in the document]
Comment (ahsonkhan) [reply xml/System.Net.Http/NSUrlSessionHandler.xml]: typo within the suggestion: valud -> value

**Comment by rpetrusha** [comment]
I've made the suggested changes, @migueldeicaza. I'll merge your PR when the build completes successfully.

**Pull request merged**